prisma/migrations/20231021213254_remove_project_matching_and_experts/migration.sql

```diff
@@ -0,0 +1,122 @@
+/* MANUAL */
+DELETE "log" WHERE 
+ "logtype" IN ('projectMatchDissolve', 'updateProjectFields', 'contactExpert');
+
+/* GENERATED */
+/*
+  Warnings:
```

Comment:
Hi @Maascha, could you clarify whether we (a) still need to keep this data somewhere (maybe as an export) for legal reasons and (b) whether there is any chance that this might be reactivated someday? If neither, I'd go ahead and drop all of this.